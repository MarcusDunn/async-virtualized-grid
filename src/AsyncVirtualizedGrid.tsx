export type AsyncVirtualizedGridProps = {

}

export const AsyncVirtualizedGrid = ({}: AsyncVirtualizedGridProps) => {
    return <></>
}