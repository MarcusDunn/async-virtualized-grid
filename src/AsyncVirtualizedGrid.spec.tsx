import { AsyncVirtualizedGrid } from "./AsyncVirtualizedGrid";
import {render} from "@testing-library/react";

test("empty async virtualized grid", async () => {
    const { container } = render(<AsyncVirtualizedGrid />);
    expect(container).toMatchSnapshot();
});