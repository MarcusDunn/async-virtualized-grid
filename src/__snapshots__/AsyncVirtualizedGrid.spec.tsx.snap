// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty async virtualized grid 1`] = `<div />`;
